core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -134,7 +134,7 @@ def install_auth_plugin
 
         Would you like to install it? (y/n)"))
 
-        @plugins_to_be_installed << 'solidus_auth_devise'
+        @plugins_to_be_installed << 'solidus_auth_devise' unless File.open('Gemfile').grep(/gem 'solidus_auth_devise'/)
```

Comment:
What about avoiding asking to install devise at all if already present?

Reply:
In what situation would someone be running the install generator but already have `solidus_auth_devise` in their Gemfile?

Reply:
@jarednorman I was under the impression the install was need to create an admin user in my environment. I had already added the solidus_auth_devise  in my dev environment because thats where the install was run. in my acceptance env, I required a new admin account and thus ran the installer again. I assume I will need to do it again in the production env

Reply:
No, that's not the case. That's done as part of loading the seeds, but the seeds can be loaded separately in environments where you need them. Once you've run the install generator in your codebase you shouldn't need to ever run it again.

Reply:
Well, I just wiped my db, ran just the migrations and the seed. Did not get a prompt for creds. So guessing there's a missing task somewhere? The default creds dont work either.

Reply:
I think there's a specific task for creating an admin user: https://github.com/solidusio/solidus_auth_devise/blob/master/lib/tasks/auth.rake#L4